JetAnalyzers/bin/jet_correction_analyzer_x.cc

```diff
@@ -18,7 +18,8 @@
 #include "CondFormats/JetMETObjects/interface/FactorizedJetCorrector.h"
 #include "PhysicsTools/Utilities/interface/LumiReWeighting.h"
 #if __has_include("xrootd/XrdCl/XrdClFileSystem.hh")
-#include "xrootd/XrdCl/XrdClFileSystem.hh"
+//#include "xrootd/XrdCl/XrdClFileSystem.hh"
```

Comment:
I left a comment in #29 about using a copy of an XRootD header. A short version is you shouldn't copy headers from external dependencies. Instead you should use static_asserts and header guards to make sure the dependencies you need exist during compilation.

Reply:
We were getting compilation errors when using the xrootd/XrdCl/XrdClFileSystem.hh file and since it’s read-only we created a slightly modified XrdClFileSystem_v2.hh script in oder to avoid the error. This was a quick and easy way to keep the workflow running without getting errors. I’m not sure I understood what you are suggesting. Could you explain more in detail?

Reply:
Without knowing what the compilation errors were I can't comment on an alternative for fixing them. I was assuming the original `xrootd/XrdCl/XrdClFileSystem.hh` file would work and that you were worried about dependencies. Perhaps you can post the compilation error you were seeing (as an attachment).

Reply:
Thank you! I am attaching an image of the error we were having. Basically we changed the kXR_evict to kXR_wait , as the error suggested, and the compilation of XrdClFileSystem_v2.hh works fine without issues.
![compilation_error](https://user-images.githubusercontent.com/54803536/98269557-87460300-1f96-11eb-80b9-f4ac3980465e.png)

Reply:
I can't be certain, but it look like adding `#include "xrootd/XProtocol/XProtocol.hh"` above the other XRootD header will fix the problem. Give it a try and let us know.

Reply:
Thank you! I tested it and it works, I don't get the compilation error anymore. I'll add this line and remove the XrdClFileSystem_v2.hh code, if that's okay with you.